### PR TITLE
RFC: Embedding -> EmbeddingModel

### PR DESCRIPTION
**Pull request by dev2049**
Embeddings sounds like the output of an embedding model, not an embedding model itself 🌝

### CODE REVIEW

File: langchain/embeddings/base.py [hunk 1]
```diff
@@ -13,3 +13,7 @@ def embed_documents(self, texts: List[str]) -> List[List[float]]:
     @abstractmethod
     def embed_query(self, text: str) -> List[float]:
         """Embed query text."""
+
+
+# For backwards compatibility.
+Embedding = EmbeddingModel
```
Comment (eyurtsev) [inline langchain/embeddings/base.py:19]: Ah good!

**Comment by eyurtsev** [comment]
@dev2049  this rename looks great to me! And it's backwards compatible soooo :)

**Comment by nfcampos** [comment]
This is great!

**Review by nfcampos — APPROVED** [review]

File: langchain/embeddings/base.py (hunk 1 above)
Comment (dev2049) [reply langchain/embeddings/base.py:19]: could even give this an init that logs deprecation warning

**Pull request closed without merging**